chain/chain/src/chain.rs

```diff
@@ -979,7 +979,9 @@ impl Chain {
 
             let protocol_version =
                 self.epoch_manager.get_epoch_protocol_version(header.epoch_id())?;
-            if ProtocolFeature::ChunkEndorsementsInBlockHeader.enabled(protocol_version) {
+            #[allow(deprecated)]
+            if ProtocolFeature::_DeprecatedChunkEndorsementsInBlockHeader.enabled(protocol_version)
```

Comment:
While working with deprecated protocol features, always assume protocol_version is greater than the feature protocol version, i.e. the feature is always enabled.